Description: Use assert_called_once_with in test_cleanup_cleanwait_timeout
Author: Corey Bryant <corey.bryant@canonical.com>
Bug: https://bugs.launchpad.net/ironic/+bug/1592852
Forwarded: https://review.openstack.org/#/c/330001/

---
 ironic/tests/unit/conductor/test_utils.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ironic/tests/unit/conductor/test_utils.py b/ironic/tests/unit/conductor/test_utils.py
index 35d384a..5c5677b 100644
--- a/ironic/tests/unit/conductor/test_utils.py
+++ b/ironic/tests/unit/conductor/test_utils.py
@@ -599,7 +599,8 @@ class ErrorHandlersTestCase(tests_base.TestCase):
         conductor_utils.cleanup_cleanwait_timeout(self.task)
         self.assertEqual({}, self.node.clean_step)
         self.assertNotIn('clean_step_index', self.node.driver_internal_info)
-        self.task.process_event.assert_called_once()
+        self.task.process_event.assert_called_once_with('fail',
+                                                        target_state=None)
         self.assertTrue(self.node.maintenance)
         self.assertEqual(clean_error, self.node.maintenance_reason)
 
-- 
2.8.1

